src/mngt/ocf_mngt_cache.c

```diff
@@ -2344,11 +2344,11 @@ ocf_promotion_t ocf_mngt_cache_promotion_get_policy(ocf_cache_t cache)
 {
 	ocf_promotion_t result;
 
-	ocf_metadata_start_shared_access(&cache->metadata.lock);
+	ocf_metadata_start_shared_access(&cache->metadata.lock, 0);
```

Comment:
`lock_idx = ocf_queue_get_next_shard_idx(cache-mngt_queue);` ?

Reply:
I think this is so rare that it isn't worth the effort